tip-wlan/values.yaml

```diff
@@ -37,11 +37,13 @@ global:
   nodePortPrefix: 302
   nodePortPrefixExt: 304
 
-  repository: tip-tip-wlan-cloud-docker-repo.jfrog.io
+  repository:
+    registry: tip-tip-wlan-cloud-docker-repo.jfrog.io
+    username: tip-read
+    password: tip-read
```

Comment:
Please don't tell me that this is the actual password. :sweat_smile:

Reply:
it is for the read-only account (as far as I'm aware). Its available in plain text in many other places, so I dont think its an issue. I've verified in jFrog that it can only read stuff.

Reply:
Don't get me wrong, I'm not concerning about you adding it here. It's just the password itself. :smile: